src/checkov/checkovRunner.ts

```diff
@@ -23,7 +24,14 @@ const getDockerFileMountParams = (mountDir: string, filePath: string | undefined
     return ['-v', `"${absPath}:${mountDir}/${baseName}"`];
 };
 
-const getDockerRunParams = (workspaceRoot: string | undefined, filePath: string, extensionVersion: string, configFilePath: string | undefined, checkovVersion: string | undefined, prismaUrl: string | undefined, certPath: string | undefined, debugLogs: boolean | undefined) => {
+const getPathParamsForDockerRun = (mountDir: string, filePath: string | undefined, flag: string): string[][] => {
+    const dockerParams = getDockerFileMountParams(mountDir, filePath);
+    const checkovParams = filePath ? [flag, `"${mountDir}/${path.basename(filePath)}"`] : [];
+
+    return [dockerParams, checkovParams];
+};
+
+const getDockerRunParams = (workspaceRoot: string | undefined, filePath: string, extensionVersion: string, configFilePath: string | undefined, checkovVersion: string | undefined, prismaUrl: string | undefined, extChecksDir: string |undefined, certPath: string | undefined, debugLogs: boolean | undefined) => {
```

Comment:
instead of using `| undefined` you can use optional `?`

Reply:
This is the standard across the whole code in this repo for similar values, so just followed it to keep convention. You think I should changed it?

Reply:
hmm I think that is righter but it will cause breaking of existing standards, 
Don't think that's worth it, let it be 😄